rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -8,25 +8,40 @@ import cdm.base.staticdata.party.*
 import cdm.observable.asset.*
 import cdm.product.template.*
 
+choice Asset:  <"An Asset is defined something that can be owned and transferred in the financial markets. As a choice data type, one and only one of the  attributes must be used. Cash: An Asset that consists solely of a monetary holding in a currency. Commodity: An Asset comprised of raw or refined materials or agricultural products, eg gold, oil or wheat. DigitalAsset An Asset that exists only in digital form, eg Bitcoin or Ethereum; excludes the digital representation of other Assets. Instrument: An asset that is issued by one party to one or more others; Instrument is also a choice data type.">
```

Comment:
having an odd feeling of deja vu with this comment... see comment I made on #3022